frameworks/digital-outcomes-and-specialists/questions/brief-responses/dayRate.yml

```diff
@@ -14,7 +14,7 @@ validations:
     message: 'You need to answer this question.'
   - name: not_money_format
     field: dayRate
-    message: "Day rate must be in numbers only, no symbols, eg 99.95"
+    message: "Day rate must be in numbers and not include symbols, for example 99.95."
```

Comment:
I see what you mean.

This is probably clearer: "Day rate must be in numbers and decimal points only, for example 99.95."

Would anyone start trying to include lots of decimal points? 

If so, maybe this: "Day rate can only include numbers and a decimal point, for example 99.95."